packages/catalog-client/src/index.ts

```diff
@@ -21,4 +21,6 @@
  */
 
 export { CatalogClient } from './CatalogClient';
-export * from './types';
+// export * from './types';
+
+export * from './generated/models';
```

Comment:
Do we want to update this to an alpha export or a new package? I'd like to export the models (and possibly an operation format for input data) from the generated code for consumption in other packages, but not sure if that should continue to exist in catalog client.